recipes/libgcc/meta.yaml

```diff
@@ -0,0 +1,15 @@
+package:
+  name: libgcc
+  version: 5.2.0
```

Comment:
no, because gcc is repackaged from the system's gcc.

GCC 5 libstdc++ has both gcc4 and gcc5 symbols.  It is backwards compatible.  It is not safe to assume that of libraries built with gcc5 being consumed by gcc4 and vice-versa.